.pylintrc

```diff
@@ -34,7 +34,7 @@ max-locals=31
 max-statements=80
 max-parents=11
 max-branches=24
-max-attributes=34
+max-attributes=36
```

Comment:
instead of this, disable exception to "AEABuilder"?

Reply:
actually AEABuilder is overbloated, lets keep max-attribtes low to get remind about attributes limit